README.md

```diff
@@ -1,289 +1,828 @@
-# GraphQL for .NET - Subscription Transport WebSockets
+# ASP.NET Core GraphQL Server driven by GraphQL.NET
 
 ![License](https://img.shields.io/github/license/graphql-dotnet/server)
-
-[![Join the chat at https://gitter.im/graphql-dotnet/graphql-dotnet](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/graphql-dotnet/graphql-dotnet?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-
-[![Run code tests](https://github.com/graphql-dotnet/server/actions/workflows/test.yml/badge.svg)](https://github.com/graphql-dotnet/server/actions/workflows/test.yml)
-[![Build artifacts](https://github.com/graphql-dotnet/server/actions/workflows/build.yml/badge.svg)](https://github.com/graphql-dotnet/server/actions/workflows/build.yml)
-[![Publish release](https://github.com/graphql-dotnet/server/actions/workflows/publish.yml/badge.svg)](https://github.com/graphql-dotnet/server/actions/workflows/publish.yml)
-[![CodeQL analysis](https://github.com/graphql-dotnet/server/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/graphql-dotnet/server/actions/workflows/codeql-analysis.yml)
-
 [![codecov](https://codecov.io/gh/graphql-dotnet/server/branch/master/graph/badge.svg?token=ZBcVYq7hz4)](https://codecov.io/gh/graphql-dotnet/server)
-
-![Activity](https://img.shields.io/github/commit-activity/w/graphql-dotnet/server)
-![Activity](https://img.shields.io/github/commit-activity/m/graphql-dotnet/server)
-![Activity](https://img.shields.io/github/commit-activity/y/graphql-dotnet/server)
```

Comment:
?

Reply:
It’s pointless garbage at the top of the repository. Who cares how many commits there are in the last week, month or year? The readme didn’t start until halfway down the first page before. Let’s keep whatever of these kinds of tags fit on a single line. I put the version, coverage, and downloads, I think (?). The version is important to returning users. Coverage and downloads help indicate how supported and popular it is.

Reply:
I also put the license and repo size. License is important to commercial users. Repo size is probably rather pointless also.

Reply:
>  Who cares how many commits there are in the last week, month or year? 

You answered yourself : 

> Coverage and downloads help indicate **how supported** and popular it is.

Reply:
Okay, but this project has historically had low commits.  It does not benefit the project to show small numbers.  Let's show "4.7 million downloads" and "95% coverage", not "1 commit last week".

Reply:
This is not impressive for a project with 4.7m downloads; makes it look more like the project is dead:
![image](https://user-images.githubusercontent.com/6377684/178587652-db4b48d8-f21c-4333-9db0-b218c8420458.png)

Reply:
It's because of `master` branch. Try https://img.shields.io/github/commit-activity/w/graphql-dotnet/server/develop but I understand your thoughts. What about https://img.shields.io/github/contributors/graphql-dotnet/server ? Maybe you'll find something usefull here https://shields.io/category/activity ?

Reply:
I'm in no hurry to add more tags.  If you like, pick one more and we will add it.  Contributors is fine.

Maybe latest release date:

https://img.shields.io/github/release-date/graphql-dotnet/server?label=latest%20release

Or maybe commits since latest release:

https://img.shields.io/github/commits-since/graphql-dotnet/server/latest

Those last two seem that they might be useful for users of the library.

Reply:
Agreem let's add them.